.gitignore

```diff
@@ -137,7 +137,7 @@ aaa/
 
 **/__pycache__/
 
-**/__pydeps__/**
+**/__pypackages__/**
```

Comment:
@jan-warchol but I meant this - why do we change this and won't this break existing `__pydeps__` dirs

Reply:
Ok, I assume I can mark this as resolved?